File: api/code-samples/python/klaytnDebug/profiling/writeBlockProfile.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

file = "block.profile"

sdk = OpenSDK(host)
debug_response = sdk.debug.write_block_profile(file)

print(json.loads(debug_response.response.data))

File: api/code-samples/javascript/klaytnDebug/profilling/writeBlockProfile.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));
    
    const file = "block.profile";

    sdk.debug.writeBlockProfile(file, {}, (err, data, response) => {
        console.log(data);
    });
}
)()